src/stcal/jump/twopoint_difference.py

```diff
@@ -255,6 +281,52 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
                             if (gdq[integ, group, row, col + 1] & dnu_flag) == 0:
                                 gdq[integ, group, row, col + 1] =\
                                     np.bitwise_or(gdq[integ, group, row, col + 1], jump_flag)
+
+        # flag n groups after all jumps to account for the transient seen
+        flag_dn_threshold = [after_jump_flag_e1, after_jump_flag_e2]
+        flag_groups = [after_jump_flag_n1, after_jump_flag_n2]
+        # ensure the smallest threshold is 1st
```

Comment:
Do we really want to leave all of this unused code in here?

Reply:
No.  Deleted.